packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -424,10 +425,8 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
           )}
         </label>
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
-        <div
-          className={classnames('Combobox__input', {
-            'Combobox__input--error': hasError
-          })}
+        <TextFieldWrapper
```

Comment:
Replacing one-off wrapper implementation with reusable implementation of a styled div that applies native `input` styles on the `div` and removes them from the nested `input`.

Discussed with @scurker